WordPress/lint.xml

```diff
@@ -37,4 +37,8 @@
         <ignore path="**/generated/**" />
         <ignore path="**/strings.xml" />
     </issue>
+
+    <issue id="TypographyEllipsis" severity="warning">
+        <ignore regexp=".*values\-[a-z][a-z].*\/strings\.xml"/>
```

Comment:
Eventually, we can replace 3 dots `...` by the ellipsis `…` character in the[ script that pulls translations](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/tools/update-translations.sh). I'm not sure if there is any use case for the 3 dots in the strings file.

Reply:
I'll look into it! I think you're right